src/main/java/org/aerogear/digger/client/model/LogStreamingOptions.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2016-2017 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.aerogear.digger.client.model;
+
+import com.offbytwo.jenkins.helper.BuildConsoleStreamListener;
```

Comment:
@matzew We didn't change the package name in our fork. The POM file does point to our own release, but the package name stays the same.